src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.ts

```diff
@@ -23,7 +23,7 @@ export class TableInteractorColumnComponent {
 
   isOrganismSorting = computed(() => this.state.componentsSorting() === NavigatorComponentSorting.ORGANISM);
 
-  navigatorComponentsTA = computed(() => this.calculateTimesAppearing(this.complexes(), this.navigatorComponents()));
+  sortedNavigatorComponents = computed(() => this.calculateTimesAppearing(this.complexes(), this.navigatorComponents()));
```

Comment:
They are not really sorted in this, just preprocesed to be ready to sort . Could be sortingReadyComponents